parser-typechecker/src/Unison/Codebase.hs

```diff
@@ -23,7 +23,9 @@ import Unison.Codebase.Editor.RemoteRepo (ReadRemoteNamespace, WriteRepo)
 import Unison.Codebase.GitError (GitError)
 import Unison.Codebase.Patch (Patch)
 import qualified Unison.Codebase.Reflog as Reflog
+import qualified U.Codebase.Reference as V2
 import Unison.Codebase.ShortBranchHash (ShortBranchHash)
+import qualified Unison.Codebase.SqliteCodebase.Conversions as Cv
```

Comment:
Bad idea to depend on the Sqlite conversion module. Seems like it was designed to mostly go one way.

I kinda do like the idea of converting Codebase to require V2 types but it's a big change.